user/views.py

```diff
@@ -18,7 +17,6 @@ class CreateTokenView(ObtainAuthToken):
 
```

Comment:
It's better to delete CreateTokenView and AuthTokenSerializer, since you are not using it